h2o-parsers/h2o-orc-parser/src/test/java/water/parser/ParseTestORCCSV.java

```diff
@@ -20,61 +23,65 @@
  */
 public class ParseTestORCCSV extends TestUtil {
 
-    private String[] csvFiles = {"smalldata/parser/orc/orc2csv/TestOrcFile.testDate1900.csv",
-            "smalldata/parser/orc/orc2csv/TestOrcFile.testDate2038.csv",
-            "smalldata/parser/orc/orc2csv/orc_split_elim.csv", "smalldata/parser/csv2orc/prostate_NA.csv",
-            "smalldata/iris/iris.csv", "smalldata/jira/hexdev_29.csv"};
+    private String[] csvFiles = {"smalldata/parser/orc/orc2csv/testTimeStamp.csv",
+            "smalldata/parser/orc/orc2csv/testDate1900.csv",
+            "smalldata/parser/orc/orc2csv/testDate2038.csv"};
 
-    private String[] orcFiles = {"smalldata/parser/orc/TestOrcFile.testDate1900.orc",
-            "smalldata/parser/orc/TestOrcFile.testDate2038.orc", "smalldata/parser/orc/orc_split_elim.orc",
-            "smalldata/parser/orc/prostate_NA.orc", "smalldata/parser/orc/iris.orc",
-            "smalldata/parser/orc/hexdev_29.orc"};
+    private String[] orcFiles = {"smalldata/parser/orc/testTimeStamp.orc",
+            "smalldata/parser/orc/TestOrcFile.testDate1900.orc",
+            "smalldata/parser/orc/TestOrcFile.testDate2038.orc"};
 
-    private Boolean[] forceColumnTypes = {false, false, false, true, true, true};
+    private Boolean[] forceColumnTypes = {false, false, false, false, true, true, true};
 
     @BeforeClass
     static public void _preconditionJavaVersion() { // NOTE: the `_` force execution of this check after setup
         // Does not run test on Java6 since we are running on Hadoop lib
-        Assume.assumeTrue("Java6 is not supported", !System.getProperty("java.version", "NA").startsWith("1.6"));
+        Assume.assumeTrue("Java6 is not supported", !System.getProperty("java.version",
+                "NA").startsWith("1.6"));
     }
 
     @BeforeClass
-    static public void setup() { TestUtil.stall_till_cloudsize(5); }
+    static public void setup() { TestUtil.stall_till_cloudsize(1); }
```

Comment:
I thought they will test your code for distributed version even if you set it to 1.